pkg/logs/agent_test.go

```diff
@@ -9,6 +9,7 @@ package logs
 
 import (
 	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
```

Comment:
This import is in a strange place. 
this may fail the linter. `goimports` should fix this